mailcheck/enabled.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+module.exports = {
+  name: 'mailcheck is enable or disabled',
+  hypothesis: 'mailcheck will lead to higher confirmation rate of accounts',
+  startDate: '2015-01-01',
+  endDate: '2015-11-01',
+  subjectAttributes: ['uuid', 'isMetricsEnabled', 'forceMailcheck'],
+  independentVariables: ['mailcheckEnabled'],
+  eligibilityFunction: function (subject) {
+    if (subject) {
+      if (subject.forceMailcheck === 'true') {
```

Comment:
@dannycoates  I see setting 'true' as a constant doesn't work outside of the module.exports, so I have to keep these here